tests/test_pods/bzip2.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Copyright © 2019 SUSE LLC
+#
+# pod_package_deps: bzip2, libbz2-devel, diffutils, coreutils, gcc
+# pod_timeout: 30
+#
+# Port of bzip2 test from qa-testsuites.
+
+# Autoinstall these files to temp directory:
+OPENQA_TEST_FILES="bzip2/torepack/testfile.gz bzip2/tocompile/trivial.c bzip2/topack"
+
+# Init test environment
+source openqa_lib.sh
+trap qa_fail ERR
+
+# qa_bzip2_bigfilerun.sh
+# ----------------------
+qa_start_test "qa_bzip2_bigfilerun"
+TEST_MBYTES="10"
+echo "Starting compression of $TEST_MBYTES MBytes .."
+dd if=/dev/urandom bs=1024 count=${TEST_MBYTES}k | bzip2 -z >/dev/null
+
+
+# qa_bzip2_bznew.sh
+# -----------------
+bznew_test () {
+	qa_start_test "qa_bzip2_bznew"
```

Comment:
I think a more common style for shell scripts (in this repo) is just 4 spaces for indendation

Reply:
This repo has a grand total of ONE shell script in `scripts/`. There's also a bunch of shell scripts in `data/` that have no common coding style to speak of. So this is a good opportunity to introduce a better coding style because space-indented code is a pain to work with.